CMakeLists.txt

```diff
@@ -52,8 +55,14 @@ if(NOT DEFINED BUILD_SHARED_LIBS)
   set(BUILD_SHARED_LIBS OFF CACHE BOOL "Global flag to cause add_library() to create shared libraries if ON")
 endif()
 
+# For now, SLEPc is always required
```

Comment:
Do you have a time in mind when this would be made optional again? If not, you could just drop the comment

Reply:
In a following PR (very soon), the complex-valued linear algebra functionality is refactored to not use PETSc and thus SLEPc will become completely optional again and this will be removed. For now, we have refactored the build system to only install PETSc for SLEPc but this isn't reflected in the Palace source yet.